mix.exs

```diff
@@ -37,7 +37,12 @@ defmodule ExImageInfo.Mixfile do
   defp aliases do
     [
       test_wip: ["test --only wip"],
-      docs: ["docs", &copy_doc_to_docs/1]
+      docs: ["docs", &copy_doc_to_docs/1],
+      lint: [
+        "format --check-formatted",
```

Comment:
Format fails because the `.formatter.exs` is not present at the project root level. And if you add one, it causes a ton of file churn. Not sure if we want to keep this.

Reply:
If the current indentation and formatting is good and clear, we can keep it like as is. If at some point you consider that would be an improvement, by applying an automatic formatter (like styler), go for it.

Reply:
I think we can probably drop it. It'll add file churn that will muddy the blame up.